Sources/SQLKit/SQLDialect.swift

```diff
@@ -9,6 +9,8 @@ public protocol SQLDialect {
     var supportsIfExists: Bool { get }
     var supportsAutoIncrement: Bool { get }
     var enumSyntax: SQLEnumSyntax { get }
+    var supportsDropTriggerTable: Bool { get }
```

Comment:
Adding default values for these two new additions would make this a non-breaking change. 👍